doc/release-notes-1.9.rst

```diff
@@ -10,8 +10,8 @@ We are pleased to announce the release of Zephyr kernel version 1.9.0
 
 Major enhancements planned with this release include:
 
-* POSIX API Layer
-* BSD Socket Support
+* Pthreads compatible API
+* BSD Sockets compatible API
```

Comment:
In that case, maybe this should read:

     * Subset of BSD Sockets compatible API